_data/en/settings.yml

```diff
@@ -156,6 +156,7 @@ help_subpages:
   specific-agencies: Help with specific agencies
   verify-your-identity: Verify your identity
 nav:
+  about_login_gov: About Login.gov
```

Comment:
We need the translations for these. Thats why tests are failing.

Reply:
Yeah the partners site doesn't require translations, so I'll just add english strings?

Reply:
> Yeah the partners site doesn't require translations, so I'll just add english strings?

Seems okay to me, if it works. Maybe add a comment in the YAML explaining why the French and Spanish strings are in English?

Also, I say "if it works", since in the IdP, we have tests to ensure that the texts are different between the language, but I don't know that those exist here.

Reply:
@mdiarra3 i put in english fillers for the translations for now. Is that good enough?

Reply:
yup all good.